### PR TITLE
multi: running content edits+additions

### PR DESCRIPTION
**Pull request by Roasbeef**
This is a running draft PR of edits/additions so far with what's currently checked into the develop branch. I've tried to make the changes in small commits which modify only a paragraph or two so it's easy to review, and also to drop/modify edits if needed. 

Leaving this in draft until this initial pass is complete. As discussed offline, this may also grow to include a more detailed outline/table-of-contents to help guide contributors as well as provide place holders for the various sections we'll write.

### CODE REVIEW

File: 01_introduction.asciidoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ While the bulk of this book is written for programmers, the first two chapters a
...
+Because blockchains are global-broadcast gossip protocols, each node is required to know and validate every single transaction that occurs on the network. Furthermore, once validated, each transaction and block must be propagated to the node's "neighbors", multiplying the bandwidth requirements. As such, the greater the block size, the greater the bandwidth, processing, and storage requirements for each individual nodes, effectively limiting the amount of scaling that can be done this way. Furthermore, scaling in this way has an undesirable side effect of centralizing the network by reducing the number of nodes and node operators. Since node operators are not compensated for running nodes, if nodes are very expensive to run, only a few well funded node operators will continue to run nodes.
```
Comment (PatrickZGW) [inline 01_introduction.asciidoc:20]: For each individual node**s** -> For each individual node

File: 01_introduction.asciidoc [hunk 2]
```diff
@@ -4,9 +4,12 @@
 
 Welcome to Mastering the Lightning Network (LN)!
 
-The Lightning Network is a protocol for using Bitcoin in a smart and non obvious way.
-Thus it is a second layer technology on top of Bitcoin.
-It is changing the way people exchange value online and it's one of the most exciting advancements to happen to the Bitcoin network over the past few years. Right now, LN is in its infancy. In concept it's about 5 years old, in implementation about 3 years old. We're only beginning to see the opportunities LN provides including improved privacy, speed, and scale. With core knowledge of LN, you can help shape the future of the network while building opportunities for yourself as well. Some basic knowledge about Bitcoin is assumed but can be readily acquired by reading the first two chapters of _Mastering Bitcoin_ which are available for free online.
+The Lightning Network is a second-layer protocol on top of Bitcoin that enables
+instant, low-cost Bitcoin payments. Unlike the base blockchain, the Lightning
+netowrk is able able to properly send micropayments, down to _fractions_ of a
```
Comment (PatrickZGW) [inline 01_introduction.asciidoc:9]: typo: netowrk -> network

File: 01_introduction.asciidoc [hunk 3]
```diff
@@ -24,10 +24,10 @@ But what if each node wasn't required to know and validate every single transact
...
-Users may pay each other on the Lightning Network as many times as they want, making use of the Bitcoin blockchain only in order to load bitcoin onto the network initially and to "settle", that is: remove bitcoin from the Lightning Network.
-The result is that many more Bitcoin payments can take place "off-chain", with only the initial loading and final settlement transactions needing to be validated and stored by Bitcoin nodes.
-Aside from reducing the burden on nodes, payments on the Lightning Network will be cheaper for users as they do not need to pay blockchain fees, and more private for users as they are not published to all participants of the network.
+The Lightning Network proposes a new network, a "second layer", where users can make transaction with each other in a peer-to-peer manner, without the necessity to publish a transaction to the Bitcoin blockchain every time.
+Users may transaction on the Lightning Network as many times with as many users as they want, making use of the Bitcoin blockchain only in order to load bitcoin onto the network initially and to "settle", that is: remove bitcoin from the Lightning Network.
```
Comment (PatrickZGW) [inline 01_introduction.asciidoc:28]: typo: transaction -> transact

**Comment by aantonop** [comment]
I assume this PR is obsolete, so I'm closing.

**Pull request closed without merging**